### PR TITLE
Fixes for external module

### PR DESCRIPTION
**Pull request by Premke**
With additions from commit LogosLabs/thebuggenie@6440402 now that module is external.

@zegenie Please do a thorough extraction for next external module since this one was broken.

@founderio You'll maybe face bugs in your fork due to module renaming, commit ed48986e1940ec209cb5a870455454170907bf2a should have squashed them all :)

### CODE REVIEW

**Comment by zegenie** [comment]
Sorry that the extraction was missing some details. It was done to create a starting point, not to necessarily be complete. Keep in mind that the master branch is not always stable.

Thanks for the fixes!

File: controllers/Main.php [hunk 1]
```diff
@@ -189,4 +209,26 @@ public function runListFieldvalues(framework\Request $request)
...
 
+    public function runIssueEditTimeSpent(framework\Request $request)
+    {
+        try
+        {
+            \thebuggenie\core\framework\Context::performAction(
+                new \thebuggenie\core\modules\main\controllers\Main(),
+                'main',
+                'IssueEditTimeSpent'
+            );
```
Comment (zegenie) [inline controllers/Main.php:220]: This is not good, at all - we'll have to revisit this soon.

Comment (Premke) [inline controllers/Main.php:220]: Lets revisit it here #3.

**Comment by founderio** [comment]
@Premke Ah, yes, I have seen some of them. But I seem to have missed the ones in cli/ as I did not even look at the CLI commands yet. That's for a later date :)
Thanks a bunch!

**Pull request merged**